test.cpp

```diff
@@ -0,0 +1,19 @@
+#include "Pusher.h"
+
+int main()
+{
+    Pusher pusher("ck.pem", "123456");

```

Comment:
I'm not sure this will work since we don't have .pem file here, hence I don't think you would like to expose your certificate file online : )